Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.py

```diff
@@ -4370,19 +4391,44 @@ def aggregated_list_instances_ip(args: Dict[str, Any]) -> CommandResults:
         CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains instance
         details.
     """
-    project = SERVICE_ACT_PROJECT_ID
     ip = args.get('ip')
+    default_search_scope = demisto.params().get('default_search_scope')
```

Comment:
Do you feel like this needs any error handling here or in the `if/else` for what the potential input could be?